stingray/largememory.py

```diff
@@ -266,11 +268,11 @@ def saveData(data, dir_name=randomNameGenerate()):
     elif isinstance(data, EventList):
         _saveChunkEV(data, dir_name, chunks)
 
-    elif os.stat(data).st_size > 0:
+    elif os.path.exists(data) and os.stat(data).st_size > 0:
```

Comment:
os.stat by default returns a FileNotFoundError so os.path.exists is redundant

Reply:
Yes, but if data are anything else than a valid file name or Eventlist or Lightcurve, it should go to the next option (we might add something else later). 
Here I'm saying: if it's a valid file name, then check the size, and if it's larger than 0 process.

Reply:
To be clear, in tests I give a string as an example of invalid value. Your version interpreted it as a file name and tried to open it. It still failed, but for the wrong reason.

Reply:
Ohh okay!